### ISSUE TITLE
Error trying to run packer.exe command

### ISSUE THREAD

**Issue by mikedouglasdev** [Microsoft/vsts-image-generation#75]
I'm trying to use this to create build VM.  The "Create required Azure Resources" section completes successfully.  However, when I run the packer.exe command in the following section, I get this error.  I ran the script from this folder \vsts-image-generation-master\images\win.  Let me know if there is something different I can try.  Thanks!

`vhd: Creating Azure Resource Manager (ARM) client ...Build 'vhd' errored: azure.BearerAuthorizer#WithAuthorization: Failed to refresh the Token for request to https://management.azure.com/subscriptions/1111111-3abd-4eac-943d-67145e4c25fb/resourceGroups/mdvs2017img/providers/Microsoft.Storage
/storageAccounts/mdvs2017img/listKeys?api-version=2016-12-01: StatusCode=0 -- Original Error: adal: Refresh request failed. Status Code = '401'
`

**Comment by chrisrpatterson** [comment]
That is an error from packer and Azure directly and not specific to these templates.  My guess is you do not have all of the appropriate variables set for your service principal does not have all of the necessary permissions.

**Comment by lokesh755** [comment]
Did you look into the doc (https://github.com/Microsoft/vsts-image-generation/blob/master/docs/vs2017.md )?  Hope it's helpful.

**Comment by mikedouglasdev** [comment]
Chris, thanks.  I'll take a look again.  I created a new AAD user with global administrator rights to AAD and subscription owner.  The "Create required Azure resources" script ran without errors but the packer gave me this error.  I agree about the error but I'm not sure where to look.  I ran it from another Azure VM (to use to run it for the 8+ hours) but I wouldn't think it would matter.  I'll try it from my laptop.

Lokesh755, thanks for sharing this but yes, this is what I followed to get the error.

Thanks,
Mike
